packages/keyed/dev/entries.tsx

```diff
@@ -0,0 +1,106 @@
+// changes to this file might be applicable to similar files - grep 95DB7339-BB2A-4F06-A34A-25DDF8BF7AF7
+
+import { createStore, produce } from "solid-js/store";
+import { createEffect } from "solid-js";
+import { Entries } from "../src/index.js";
+import { TransitionGroup } from "solid-transition-group";
+
+const foods = [
+  "oatmeal",
+  "plantains",
+  "cranberries",
+  "chickpeas",
+  "tofu",
+  "Parmesan cheese",
+  "amaretto",
+  "sunflower seeds",
+  "grapes",
+  "vegemite",
+  "pasta",
+  "cider",
+  "chicken",
+  "pinto beans",
+  "bok choy",
+  "sweet peppers",
+  "Cappuccino Latte",
+  "corn",
+  "broccoli",
+  "brussels sprouts",
+  "bread",
+  "milk",
+  "honey",
+  "chips",
+  "cookie",
+];
+const randomIndex = (list: readonly any[]): number => Math.floor(Math.random() * list.length);
+const getRandomFood = () => foods[randomIndex(foods)]!;
+const randomKey = (record: Record<string, string>): string => {
+  const keys = Object.keys(record);
+  return keys[randomIndex(keys)]!;
+};
+
+export default function App() {
+  const [store, setStore] = createStore<{ entries: Record<string, string> }>({
+    entries: {
```

Comment:
This could just be a signal `const [entries, setEntries] = createSignal({...})`

Reply:
I feel like this should remain a `store` because it's an object which Solid natively tracks, and making it a signal would kinda make the "Clone" button pointless (or more pointless than it already is, hah).

Reply:
youre right lets leave it